docker/quickstart.requirements.txt

```diff
@@ -1 +1,2 @@
 datasets
+rq == 1.15.1
```

Comment:
We can remove this from here.

We have this requirements file just to install the `datasets` required by the `docker/scripts/load_data.py` of the Docker quickstart image.

Reply:
As far as I understand these are the requirements of the image so I don't see any problem adding another requirement. Remember that we need `rq` to run the worker that will consume the enqueued jobs.